charts/newrelic-infrastructure-v3/values.yaml

```diff
@@ -116,11 +117,14 @@ ksm:
       memory: 150M
   config: {}
 
-# -- Configuration for the DaemonSet that collects metrics from the control plane.
+# controlPlane -- Configuration for the DaemonSet that collects metrics from the control plane.
```

Comment:
theoretically it could be also a deployment since we expose the type

Reply:
Fixed!